OracleDatabase/README.md

```diff
@@ -12,6 +12,12 @@ To assist in building the images, you can use the [buildDockerImage.sh](dockerfi
 
 The `buildDockerImage.sh` script is just a utility shell script that performs MD5 checks and is an easy way for beginners to get started. Expert users are welcome to directly call `docker build` with their prefered set of parameters.
 
+### Requirements for Building
+If you intend to build, make sure you have:
```

Comment:
I would rewrite this to: "in order to successfully build this image, please ensure your build host has:"